engine/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apt-get update && apt-get install -y \
     gcc \
     libmariadb-dev \
     libpq-dev \
-    netcat \
+    netcat-traditional \
```

Comment:
For Debian 12, it's required to specify `netcat-traditional` vs `netcat-openbsd` explicitly. Previously we used `netcat-traditional`, so leaving it as it.